Source/Core/Core/Boot/Boot_BS2Emu.cpp

```diff
@@ -229,7 +229,7 @@ bool CBoot::SetupWiiMemory(DiscIO::IVolume::ECountry country)
 
 		if (!settingsFileHandle.WriteBytes(gen.GetData(), SettingsHandler::SETTINGS_SIZE))
 		{
-			PanicAlertT("SetupWiiMemory: Cant create setting.txt file");
+			PanicAlertT("SetupWiiMemory: Can't create setting.txt file");
```

Comment:
I'm still not entirely sure if a string gets completely wiped or not. If it does get wiped, I'll remove this from the PR.